pkg/storage/unified/search/document.go

```diff
@@ -43,6 +43,18 @@ func (s *StandardDocumentBuilders) GetDocumentBuilders() ([]resource.DocumentBui
 				dsinfo = append(dsinfo, info)
 			}
 		}
+
+		// Fetch dashboard sprinkles for the namespace
+		// This could take a while if namespace has a lot of dashboards
+		var stats map[string]map[string]int64
+		if s.sprinkles != nil {
+			stats, err = s.sprinkles.GetStats(ctx, namespace)
+			// Should we just log a warning here if it fails? Seems overkill to return an error for sprinkles and fail the index startup
```

Comment:
@scottlepp Should we just fail with a warning log for sprinkles?

Reply:
I would log an error so at least we get alerted, but doesn't seem like we have to fail the startup.